examples/SharpBrick.PoweredUp.Examples/ExampleTwoPortHubMediumLinearMotor.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Threading.Tasks;
+using SharpBrick.PoweredUp;
+
+namespace Example
+{
+    public class ExampleTwoPortHubMediumLinearMotor : BaseExample
```

Comment:
I usually add the test run to the Program.cs commented out ... like that it can be easier discovered. Fix it on the next PR.

Reply:
Ok, good idea! Will take care of that in the next PR